mpas_analysis/ocean/climatology_map_waves.py

```diff
@@ -0,0 +1,526 @@
+    """
+    A subtask for reading and remapping wave climatology
+    """
+    # Authors
+    # -------
+    # Steven Brus, Xylar Asay-Davis
+
+    def customize_masked_climatology(self, climatology, season):  # {{{
+        """
+        Convert peak freqency to peak period
+
+        The descriptor for the observation grid
+        Parameters
+        ----------
+        climatology : ``xarray.Dataset`` object
+        the climatology data set
+
+        season : str
+        The name of the season to be masked
+
+        Returns
+        -------
+        climatology : ``xarray.Dataset`` object
+        the modified climatology data set
+        """
+        # Authors
+        # -------
+        # Steven Brus
+        # Xylar Asay-Davis
+
+        if 'timeMonthly_avg_peakWaveFrequency' in climatology:
+            climatology['peakWavePeriod'] = \
+                1.0/climatology['timeMonthly_avg_peakWaveFrequency']
+
+        if 'timeMonthly_avg_significantWaveHeight' in climatology:
+            climatology['significantWaveHeight'] = \
+                climatology['timeMonthly_avg_significantWaveHeight']
+
+        return climatology
+
+        # }}}
+    # }}}
+
+
+class RemapEra5ObservedWaveClimatology(RemapObservedClimatologySubtask):  # {{{
+    """
+    A subtask for reading and remapping ERA5 wave observations
+    """
+    # Authors
+    # -------
+    # Steven Brus, Xylar Asay-Davis
+
+    def __init__(self, parentTask, seasons, fileName, outFilePrefix,
+                 variableList, subtaskName,
+                 comparisonGridNames=['latlon']):
+        # {{{
+        '''
+        Construct one analysis subtask for each plot (i.e. each season and
+        comparison grid) and a subtask for computing climatologies.
+
+        Parameters
+        ----------
+        parentTask :  ``AnalysisTask``
+            The parent (master) task for this subtask
+
+        seasons : list of str
+           A list of seasons (keys in ``constants.monthDictionary``) over
+           which the climatology should be computed.
+
+        fileName : str
+            The name of the observation file
+
+        outFilePrefix : str
+            The prefix in front of output files and mapping files, typically
+            the name of the field being remapped
+
+        variableList : list
+            List of observational variables to remap
+
+        comparisonGridNames : list of {'latlon', 'antarctic', 'arctic'},
+            optional
+            The name(s) of the comparison grid to use for remapping.
+
+        subtaskName : str, optional
+            The name of the subtask
+        '''
+        # Authors
+        # -------
+        # Steven Brus
+
+        self.variableList = variableList
+
+        # call the constructor from the base class (AnalysisTask)
+        super().__init__(parentTask, seasons, fileName, outFilePrefix,
+                         comparisonGridNames=['latlon'],
```

Comment:
Because this is hard coded, only `latlon` works right now.  I think you want:
```suggestion
                         comparisonGridNames=comparisonGridNames,
```

Reply:
Thanks for catching this.